etna/core/blocks/document.py

```diff
@@ -0,0 +1,53 @@
+from wagtail import blocks
+from wagtail.documents.blocks import DocumentChooserBlock
+
+
+class DocumentBlock(blocks.StructBlock):
+    """
+    A block for embedding a document file in a page.
+    """
+
+    link_text = blocks.CharBlock(
+        required=True,
+        help_text="The text to display for the link to the document",
+    )
```

Comment:
Not sure we actually need this anymore... Could this just be the file title? We have "file link text" in the requirements, but in the designs we are basically just using the title
![image](https://github.com/nationalarchives/ds-wagtail/assets/62654785/e0f46fb4-8a3d-484a-85c4-11fb4e114c01)
![image](https://github.com/nationalarchives/ds-wagtail/assets/62654785/f109440b-8d20-41db-ba60-843ccd0f1178)

Reply:
Yes - we should just use the document title 👍🏼

Reply:
(as long as you can change the name of the file after uploading to Wagtail)